zboxcore/sdk/filemetaworker.go

```diff
@@ -27,60 +24,42 @@ type fileMetaResponse struct {
 
 func (req *ListRequest) getFileMetaInfoFromBlobber(blobber *blockchain.StorageNode, blobberIdx int, rspCh chan<- *fileMetaResponse) {
 	defer req.wg.Done()
-	body := new(bytes.Buffer)
-	formWriter := multipart.NewWriter(body)
 
 	var fileRef *fileref.FileRef
 	var s strings.Builder
 	var err error
+	grpcReq := &blobbergrpc.GetFileMetaDataRequest{
+		Allocation: req.allocationTx,
+	}
```

Comment:
Is there a convenient way of mocking `blobbergrpc` functions for unit tests? For instance PR https://github.com/0chain/gosdk/pull/48 has a unit test for `getFileMetaInfoFromBlobber` and will need to merge this change in.

Reply:
will need to run mockery in blobber to generate mocks for https://github.com/0chain/blobber/blob/a73ef1703eb84fea60bfe48e5771b99978ec13b0/code/go/0chain.net/blobbercore/blobbergrpc/blobber_grpc.pb.go#L19
Then we import `blobbercore/mocks` here

Reply:
@Sriep

Reply:
Looks ok. Do we need this merged?

Reply:
Its being merged to grpc not master. There will be a seperate PR between master<>grpc so issues will be caused then - ofcourse current grpc branch will be tested on a devnet before we merge it on master.

I need to make some go.mod changes before i merge it so - if you approve this changes please add your approval, thank you!

Reply:
Added an issue - https://github.com/0chain/blobber/issues/176